src/coffee/services/product-projections.coffee

```diff
@@ -250,7 +250,6 @@ class ProductProjectionService extends BaseService
   #   .search()
   search: ->
     @_currentEndpoint = '/product-projections/search'
-    @fetch()
```

Comment:
I would vote to break here in order to not get to much confused.
The API is clearer afterwards.

Bump minor version + inform on release nodes + in the JSDoc.